vaccine_feed_ingest/runners/us/physicians_immediate/normalize.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import pathlib
+import pprint
+import sys
+from typing import Dict, List, Optional, OrderedDict
+
+import usaddress
+from vaccine_feed_ingest_schema import location as schema
+
+from vaccine_feed_ingest.utils.log import getLogger
+
+logger = getLogger(__file__)
+
+Site = Dict[str, str]
+
+
+def get_site_id(site: Site) -> str:
+    return f"us_physicians_immediate:{site['table_id']}_{site['row_id']}"
+
+
+def infer_address(site: Site) -> schema.Address:
+    """The source data does not separate address components into separate fields.
+    This function imports the probabilistic parser from `usaddress` to make sense
+    of the scraped address data. Some extra patches are applied to fix errors in this
+    specfic dataset.
+    """
+
+    parsed_address, address_type = usaddress.tag(site["address"])
+    assert (
+        address_type == "Street Address"
+    ), f"Invariant: address normalization only works for addresses of type 'Street Address', not {address_type}"
```

Comment:
Should we handle the case where these assertion fails (potentially with a `try`/`except` in the outer loop) so that it drops only individual bad records, rather than fails processing for the entire data source?